### PR TITLE
Work in progress for the journal

### PR DESCRIPTION
**Pull request by JohnPWSharp**
Incorporated feedback from @bennage. Started work on journal entry 1

connects to #123

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@JohnPWSharp__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: docs/journal/01-cold-storage.md [hunk 1]
```diff
@@ -3,66 +3,213 @@ _Capturing event data and saving it in its raw format to cold storage_
...
+> The purpose of this phase is to determine an approach to ingesting data, performing the simplest of processing, and saving it for subsequent analysis. Because the customer requires all event data to be stored indefinitely, the volume of data held in cold storage could become very large. Cold storage must therefore be inexpensive.
+
+# Fabrikam's Perspective
 Fabrikam's engineering team has very little experience with high-scale event-oriented systems. They want to make sure that they are [asking the right questions][orientation]. They also need to get a minimum viable product deployed.
 
 For their [first milestone][milestone] they want to:
 - Ingest a representative set of simulated events.
 - Store the raw data for all events for later analysis. [:arrow_right:][cold-storage]
-- Increase the scale until they match the targets for there first contract [:arrow_right:][increase-scale]
+- Increase the scale until they match the targets for their customer [:arrow_right:][increase-scale]
```
Comment (hveiras) [inline docs/journal/01-cold-storage.md]: This arrow is not rendered properly

**Comment by hveiras** [comment]
It is fine by me. Anyone consider any of the TBD a must to merge?

**Comment by bennage** [comment]
We can leave the TBD for later.

**Comment by bennage** [comment]
@JohnPWSharp this is described as "work in progress", but I think that we can merge. Are you okay with that?

File: docs/journal/01-cold-storage.md (hunk 1 above)
Comment (bennage) [inline docs/journal/01-cold-storage.md]: Odd. It renders here: https://github.com/mspnp/iot-journey/blob/master/docs/journal/01-first-steps.md
Perhaps this is just a bug in the rendering of the pr? :confused:

**Comment by JohnPWSharp** [comment]
Fine with me

**Pull request merged**